RELEASING.md

```diff
@@ -1,12 +1,12 @@
 # Creating a new release
 
-1. Prep docs PR for updated layer version and release version. **NOTE**: layer version and release version are totally different :(
 1. Update `version.go` with new layer version.
 1. Update `README.md` with new layer version.
 1. Add new entry in `CHANGELOG.md`, include both release version and layer version.
 1. Open a PR for release prep.
 1. Once the above changes are merged into `main`, tag `main` with the new release version, e.g. `v0.1.1`. Push the tag. This will kick off CI, which will create a draft GitHub release and publish the new layer version in AWS.
 1. Update release notes on the new draft GitHub release with notes from changelog and Layer Version ARN, and publish that.
+1. Prep docs PR for updated layer version and release version. **NOTE**: layer version and release version are totally different :(
```

Comment:
moved this lower in the instructions because it's easier if you know what version you're bumping to before updating docs

Reply:
I think we may want to keep this as the first step, as it was done intentionally because we kept missing it 🙈 

Instead we may still need to add an extra step at the end to finalize the docs PR once the version number is known? Alternatively we may want to add a release workflow like we did for refinery, to open a new issue on our internal board to update docs...

Reply:
A workflow to create the Docs issue/PR would be great.

Maybe we have remainders both at beginning and end of releasing steps to help remember 😁

Reply:
As a compromise, I've moved it to the second step so that it's after we update the lambda layer version but it's still top of mind! Let me know if that works for y'all otherwise I'm happy to move it back.